timemachine/fe/topology.py

```diff
@@ -292,6 +329,25 @@ def parameterize_periodic_torsion(self, proper_params, improper_params):
         combined_potential = potentials.PeriodicTorsion(combined_idxs, combined_lambda_mult, combined_lambda_offset)
         return combined_params, combined_potential
 
+    def setup_end_state(self, ff):
```

Comment:
Should this just be 
`setup_end_state(self)` and use `self.ff` here?

It seems odd to pass in a (possibly different) ff here.

Reply:
good call - goof'd on my end